src/main/java/de/zalando/ep/zalenium/container/kubernetes/KubernetesContainerClient.java

```diff
@@ -414,6 +419,26 @@ public boolean isReady(ContainerCreationStatus container) {
         }
     }
 
+    public PodStatus getPodStatus(ContainerCreationStatus container) {
```

Comment:
What is the purpose of this `getPodStatus` method? I see it being used in the `DockeredSeleniumStarter.java` class, but the method `getPodStatus` over there is not called anywhere else.

Reply:
Yeah, I think that now I can remove it, it's only for debug. I'll remove it.

Reply:
About the unit tests, I realized that the problem is because in the test environment the pods doesn't have pod phase. So now, with the new running phase check, the registerNode fail.

We added this running phase check because a selenium-nodes (pods) can try to register   before the eks API Server knows he is in Running Phase what later will create other problems, i.e. in checkContainers. If the registration process fail the selenium-node automatically is going to try to register later so this check solve that problem.

Do you know more about how to avoid this problem in the Test environment? If you want, we can try to talk about this by phone.